Source/Controllers/DataControllers/SPTableData.m

```diff
@@ -437,6 +437,82 @@ - (BOOL) updateInformationForCurrentView
 	return YES;
 }
 
+/**
+ * Retrieve the CREATE statement for a table/view and return extracted table
+ * structure information.
+ * @attention This method will interact with the UI on errors/connection loss!
+ */
+- (NSDictionary *) informationForTable:(NSString *)tableName fromDatabase:(NSString *)database
```

Comment:
This is a lot of duplicated code from the standard informationForTable method below. Could you make informationForTable call this with the database name of the current database to make them share all the rest of the code?
In other words, informationForTable just grabs the name of the current DB and then calls informationForTable:fromDatabase:

Reply:
Should be able to get current DB name from [tableDocumentInstance database] or similar

Reply:
Yes, I forgot it, but initially I intended to simplify the 2 methods, I will modify it.